ProjectSystemTools.sln

```diff
@@ -1,13 +1,11 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26913.0
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30421.23
```

Comment:
Where is this version number coming from?

Reply:
Should be changed back now, didn't seem to affect the repo